deploy/examples/import-external-cluster.sh

```diff
@@ -298,6 +299,63 @@ eof
   fi
 }
 
+function getTopologyTemplate() {
+  topology=$(
+    cat <<-END
+     {"poolName":"$1",
+      "domainSegments":[
+        {"domainLabel":"$2","value":"$3"}]},
+END
+  )
+}
+
+function createTopology() {
+  TOPOLOGY=""
+  declare -a topology_failure_domain_values_array=()
+  declare -a topology_pools_array=()
+  topology_pools=("$(echo "$TOPOLOGY_POOLS" | tr "," "\n")")
+  for i in $topology_pools; do topology_pools_array+=($i); done
+  topology_failure_domain_values=("$(echo "$TOPOLOGY_FAILURE_DOMAIN_VALUES" | tr "," "\n")")
```

Comment:
any other way to split a comma separated list to array in bash script

Reply:
This snippet sets the internal field separator to `,`, then uses `read -a` to read into the array named `fields`.
```sh
IFS=, read -a fields <<< "$TOPOLOGY_FAILURE_DOMAIN_VALUES"
```

Reply:
@parth-gr do you need to use those values later or you just want to iterate over them?

Reply:
@renan-campos I just want to iterate them and call them as a function paramter